<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuConfig</class>
 <widget class="QDialog" name="MenuConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Log dir:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineLogDir"/>
       </item>
       <item>
        <widget class="QPushButton" name="pbBrowseLog">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Data dir:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="lineDataDir"/>
       </item>
       <item>
        <widget class="QPushButton" name="pbBrowseData">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Events per file:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sbRootFileSize">
           <property name="maximum">
            <number>1000000</number>
           </property>
           <property name="singleStep">
            <number>10000</number>
           </property>
           <property name="value">
            <number>10000</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="chbSaveRootFile">
         <property name="text">
          <string>Save events to .root file</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QCheckBox" name="chbAdvanced">
           <property name="text">
            <string>Advanced mode</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="pbApply">
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbSaveToFile">
         <property name="text">
          <string>Save to file</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbBrowseLog</sender>
   <signal>clicked()</signal>
   <receiver>MenuConfig</receiver>
   <slot>OnBrowseLog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbApply</sender>
   <signal>clicked()</signal>
   <receiver>MenuConfig</receiver>
   <slot>OnApply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>MenuConfig</receiver>
   <slot>OnCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbSaveToFile</sender>
   <signal>clicked()</signal>
   <receiver>MenuConfig</receiver>
   <slot>OnSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbBrowseData</sender>
   <signal>clicked()</signal>
   <receiver>MenuConfig</receiver>
   <slot>OnBrowseData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnBrowseLog()</slot>
  <slot>OnBrowseData()</slot>
  <slot>OnApply()</slot>
  <slot>OnCancel()</slot>
  <slot>OnSave()</slot>
 </slots>
</ui>
